src/strings/csv/es.csv

```diff
@@ -622,6 +622,7 @@ NURSERY_REQUIRED,Vivero *
 NURSERY_TRANSFER,Traslado de vivero
 OBSERVATION_IN_PROGRESS,Observación en progreso
 OBSERVATION_STATUS,Estado de la observación
+OBSERVATION_STATUS_OUTSTANDING,Sobresaliente,In the sense of Pending or Unresolved
```

Comment:
ok this didn't work either.. we could try another comment .. cc @charlie-tf @constanzauanini

Reply:
merged the comment/redo.. let's see if this PR updates with another translation